server/features/etcd_features.go

```diff
@@ -59,18 +64,20 @@ const (
 
 var (
 	DefaultEtcdServerFeatureGates = map[featuregate.Feature]featuregate.FeatureSpec{
-		DistributedTracing:      {Default: false, PreRelease: featuregate.Alpha},
-		StopGRPCServiceOnDefrag: {Default: false, PreRelease: featuregate.Alpha},
-		InitialCorruptCheck:     {Default: false, PreRelease: featuregate.Alpha},
-		CompactHashCheck:        {Default: false, PreRelease: featuregate.Alpha},
+		DistributedTracing:           {Default: false, PreRelease: featuregate.Alpha},
+		StopGRPCServiceOnDefrag:      {Default: false, PreRelease: featuregate.Alpha},
+		InitialCorruptCheck:          {Default: false, PreRelease: featuregate.Alpha},
+		CompactHashCheck:             {Default: false, PreRelease: featuregate.Alpha},
+		TxnModeWriteWithSharedBuffer: {Default: false, PreRelease: featuregate.Alpha},
```

Comment:
Flag was enabled to true, we should mark it as enabled by default in feature flag too. We also expect flags enabled by default are Beta we should.

Reply:
yes, that's was also my confusion, I will make the changes.

Reply:
I have addressed this comment. PTAL